src/scripts/StoreTalismanApi.js

```diff
@@ -72,4 +83,111 @@ export default class StoreTalismanApi extends StoreRestApi {
 		delete this.axiosInstance.defaults.headers.Authorization;
 		await this.axiosInstance.get('/oauth/logout');
 	}
+
+	async getAllProjects() {
+		this.setSitemapPath();
+		const queryData = {
+			query: GET_ALL_PROJECTS_QUERY,
+			variables: {
+				limit: 100,
+				sortField: 'title',
+				sortDirection: 'ascending',
+			},
+		};
+		return this.axiosInstance.post('/graphql', queryData);
+	}
+
+	async getAllSitemaps(projectId) {
+		if (!projectId) {
+			projectId = this.sitemapsPath.split('/')[2];
+		}
+		this.setSitemapPath(projectId);
```

Comment:
Do not implicitly set project while accessing its sitemaps. Either set project explicitly from controller or make store methods accept project argument.